charts/stable/minecraft-java/values.yaml

```diff
@@ -74,6 +78,18 @@ service:
         enabled: true
         port: 25575
 
+mcbackup:
+  initial_delay: 2m
+  backup_interval: 24hr
+  player_online_check_interval: 5m
+  prune_backups_days: "7"
+  pause_if_no_players: false
+  link_latest: false
+  tar_compress_method: gzip
+  zstd_params: --long=25 --single-thread
+  excludes: "*.jar,cache,logs,*.tmp"
+  rcon_pass: "{{ .Values.env.RCON_PASSWORD }}"
```

Comment:
remove this, and use the `{{ }}` directly on the env

Reply:
It uses the same variable and crap out the cli, when i moved it here it became happy 🤷‍♂️